### PR TITLE
add width input to view_img()

### LINKED ISSUE #4415: [ENH] Adjust the image output size of plotting.view_img()
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe your proposed enhancement in detail.

I would like to add an input to view_img() to control the size of the output HTML image.

### Benefits to the change

The output image from view_img() is sometimes too small for exploring activation maps in detail.

### Pseudocode for the new behavior, if applicable

```python
from nilearn.plotting import view_img

view_img(<path_to_img>, width_view = 1000)

```

### PR DESCRIPTION
**Pull request by alexsayal**
- Closes #4415 

Changes proposed in this pull request:

- Add input field for the output image width of view_img() in pixels

### CODE REVIEW

File: nilearn/plotting/html_stat_map.py [hunk 1]
```diff
@@ -333,7 +333,7 @@ def _json_view_size(params):
     slices_height = 1.20 * slices_height
 
     # Get the final size of the viewer
-    width_view = 600
+    # width_view = 600
```
Comment (bthirion) [inline nilearn/plotting/html_stat_map.py:336]: Please remove the line

**Review by bthirion — COMMENTED** [review]
The change LGTM. I don't understand the codecov issue.

**Review by bthirion — APPROVED** [review]
LGTM, thx.

**Comment by man-shu** [comment]
> The change LGTM. I don't understand the codecov issue.

The coverage just updates slowly, I think. Now it is 100%, so all good!

**Review by man-shu — CHANGES_REQUESTED** [review]
Thanks for the contribution @alexsayal ! The change looks good to me too!

Since this is your first contribution could you please:

1.  add yourself to the authors section of `CITATION.cff`
2. also add this change to the `nilearn/doc/changes/latest.rst`

See https://nilearn.github.io/stable/development.html#changelog for more details, and feel free to ping me here if needed!

**Comment by man-shu** [comment]
Just made a couple of changes:
- the docs builder was failing because `view_img` can be directly imported from `nilearn.plotting` (no need to reference `html_stat_map`)
- this is probably better suited as an Enhancement in the API

**Review by man-shu — APPROVED** [review]
All good now. Thanks!

**Pull request merged**